lambda/src/main/java/com/lambda/IssueUpdater.java

```diff
@@ -57,22 +65,50 @@ public Issue setActualHours(final int issueId) {
                     LocalDateTime.now());
         }
         DecimalFormat df = new DecimalFormat("0.0");
-        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60)));
+        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60.0)));
 
         if (actualHours > 999 || actualHours < 0)
             return null;
 
         return client.updateIssue(new UpdateIssueParams(issue.getId()).actualHours(actualHours));
     }
 
+    private Duration calculateWorkingHours(LocalDateTime start, LocalDateTime end) {
+        LocalDateTime current = start;
+        Duration totalWorkingDuration = Duration.ZERO;
+
+        while (current.isBefore(end)) {
+            if (isWeekday(current)) {
+                LocalDateTime endOfDay = current.withHour(19).withMinute(30); // End at 17:30 ITC as 19:30 JST
+                
+                // Add hours for current day or until end time
+                LocalDateTime actualEnd = isSameDate(end, endOfDay) ? end : endOfDay;
```

Comment:
[ask]
If the issue was closed at 20:30 for example (after 19:30), does this logic use 19:30 as the closed time?

Reply:
this will use the time issue closed (20:30 as example)
just updated when the current day in the loop is before `end`